symfony/security-bundle/7.3/config/packages/security.yaml

```diff
@@ -0,0 +1,39 @@
+security:
+    # https://symfony.com/doc/current/security.html#registering-the-user-hashing-passwords
+    password_hashers:
+        Symfony\Component\Security\Core\User\PasswordAuthenticatedUserInterface: 'auto'
+    # https://symfony.com/doc/current/security.html#loading-the-user-the-user-provider
+    providers:
```

Comment:
I'd add a `\n` before each of the main sections of this file (`providers`, `firewalls`, etc. ) to make it easier to scan.